engine/inc/com/centreon/engine/tag.hh

```diff
@@ -31,7 +31,8 @@ class tag;
 }
 
 using tag_map =
-    absl::flat_hash_map<uint32_t, std::shared_ptr<com::centreon::engine::tag>>;
+    absl::flat_hash_map<std::pair<uint64_t, uint32_t>,
+                        std::shared_ptr<com::centreon::engine::tag>>;
```

Comment:
Why using a map?
Do you need to look for a specific tag from its id/type? I don't think so.
So, a std::list should be more appropriate.